en/developer-docs/docs/consuming-services/consuming-a-service-apikey.md

```diff
@@ -0,0 +1,49 @@
+# Consume an API Key Secured Service
```

Comment:
https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting
Here there is a mention on keeping file name and title consistent. Shall we apply that across all your docs.

Reply:
Fixed with [5843748](https://github.com/wso2/docs-choreo-dev/pull/1756/commits/584374873f731312d2f10089227696f791a33938)